samples/bluetooth/central_uart/src/main.c

```diff
@@ -289,14 +289,25 @@ static int uart_init(void)
 static void discovery_complete(struct bt_gatt_dm *dm,
 			       void *context)
 {
+	int err;
 	struct bt_nus_client *nus = context;
 	LOG_INF("Service discovery completed");
 
 	bt_gatt_dm_data_print(dm);
 
-	bt_nus_handles_assign(dm, nus);
-	bt_nus_subscribe_receive(nus);
+	err = bt_nus_handles_assign(dm, nus);
+	if (err) {
+		LOG_INF("bt_nus_handles_assign failed with errorcode %i", err);
+		goto cleanup;
+	}
+
+	err = bt_nus_subscribe_receive(nus);
+	if (err) {
+		LOG_INF("bt_nus_subscribe_receive failed with errorcode %i", err);
```

Comment:
```suggestion
		LOG_INF("bt_nus_subscribe_receive failed (err %d)", err);
```

Just for cohesion with the rest of sample logs